WebContent/tests/test21.js

```diff
@@ -0,0 +1,39 @@
+let zipFS = new zip.fs.FS();
+
+function findMaxDepth() {
+  var depth = 0;
+  try {
+    (function next() { ++depth; next(); })();
+  } catch (e) {}
```

Comment:
I showed this snippet just to show how to get the current stack limit.

To make sure that the test is deterministic, don't use a dynamic limit. Instead, pick a limit that is reasonably high, and optionally check that the limit triggers a stack overflow.